R/classes-plate.R

```diff
@@ -46,11 +44,16 @@ is_valid_data_type <- function(data_type) {
 }
 
 
-#' @title Plate
+#' Plate
+#'
 #' @description
 #' A class to represent the luminex plate. It contains information about
 #' the samples and analytes that were examined on the plate as well as
 #' some additional metadata and batch info
+#'
+#' @import R6
```

Comment:
Is it realy needed since we are using R6::R6Class anyway?

Maybe 

#' @importFrom R6 R6Class

it provides clarity as #' @import R6 does and then we can use just invoke R6Class in code

Reply:
Okey I might change that to `@importFrom R6 R6Class`